### ISSUE TITLE
{Linked:Bug9781003;Linked:Bug9768640;Freeway-Bug} [Adaptive Cards> Designer]: Screen reader not announcing any information after clicking on 'Show next/previous' buttons in date picker.

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards#8398]
### Target Platforms

Other

### SDK Version

1.5

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
OS Version: 22H2 (Build 25247.1000)
Browser: Edge Dev (109.0.1495.2)

Screen Reader: NVDA
/Narrator

**Repro Steps:**
1. Open the above URL in edge dev browser.
2. Navigate to the 'Designer' tab through tab key and activate it.
3. Navigate to the 'Sample data Editor' and ' Card payload editor' and attach the provided JSON file.
4. Navigate to 'Preview mode' and activate it.
5. Navigate to 'Enter a Date' button and activate it.
6. Navigate to the 'Show Next month' button and press enter key and observe the issue.

**Actual Result:**
Screen reader not announcing any information after clicking on 'Show next/previous' buttons in date picker.

**Expected Result:**
Screen reader should announce information after clicking on 'Show next/previous' buttons in date picker.

**Note:**
1. This issue is observed on chrome browser as well with NVDA and Narrator screen reader.
2. This issue is repro with all screen reader.

**User Impact:**
It will impact the screen reader user if the screen reader does not convey the information while activating any control on the screen as user won't be able to know whether control is activated or not?

**MAS Reference:**
https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships.html

"Have feedback to share on Bugs? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-apoorva%40microsoft.com%7c551850210d444be9b28308d9691e3dde%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656400998658942%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=l6w0SvVcMSvyGnJ6%2B6uSSgd3braF2dZ7/AdBrqqzIR8%3D&reserved=0) “.

### Screenshots

https://user-images.githubusercontent.com/95913748/227477654-162e5eb4-a747-44a2-901b-9b19554ce6e7.mp4

https://user-images.githubusercontent.com/95913748/227477499-307401f1-2b6e-40f7-ab66-e55c06aeead3.mp4

### Card JSON

```JSON
{
  "$schema": "http://adaptivecards.io/schemas/adaptive-card.json",
  "type": "AdaptiveCard",
  "version": "1.3",
  "body": [
    {
      "type": "TextBlock",
      "text": "Date Input"
    },
    {
      "type": "Input.Date",
      "id": "date",
      "placeholder": "Enter a date",
      "value": "2017-10-12"
    }
  ],
  "actions": [
    {
      "type": "Action.Submit",
      "title": "OK"
    }
  ]
}
```

**Comment by vagpt** [comment]
This issue can be tracked using the below URL.

https://microsoftit.visualstudio.com/OneITVSO/_workitems/edit/9781003

**Comment by anna-dingler** [comment]
Hi @vagpt,

This bug is a duplicate of https://github.com/microsoft/AdaptiveCards/issues/8097. As we discussed, this is was transferred to the Edge team and should not be filed on our team.

The new bug is here: https://microsoft.visualstudio.com/Edge/_workitems/edit/43722704

Please let me know if this should be reopened - thanks!

**Comment by vagpt** [comment]
Adding A11yTTValidated tag as this issue now tracked on Edge team VSO.
